docs/examples/ratify-high-availability.md

```diff
@@ -0,0 +1,141 @@
+# Install Ratify for High Availability
```

Comment:
I'm still working through how to properly setup Redis and Dapr so that they use TLS for the communication. These should be just doc updates on steps for config.

Reply:
Dapr currently doesn't support server cert verification or client ca cert usage when communicating with redis. It only supports https without cert validation.